projects/valtimo/document/src/lib/models/case-tags.model.ts

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2015-2024 Ritense BV, the Netherlands.
+ *
+ * Licensed under EUPL, Version 1.2 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://joinup.ec.europa.eu/collection/eupl/eupl-text-eupl-12
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" basis,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {TagType} from 'carbon-components-angular';
+
+enum CaseTagColor {
+  WarmGray = 'WARMGRAY',
+  Red = 'RED',
+  Magenta = 'MAGENTA',
+  Purple = 'PURPLE',
+  Blue = 'BLUE',
+  Cyan = 'CYAN',
+  Teal = 'TEAL',
+  Green = 'GREEN',
+  Gray = 'GRAY',
+  CoolGray = 'COOLGRAY',
+  HighContrast = 'HIGHCONTRAST',
+  Outline = 'OUTLINE',
+}
+
+interface CaseTag {
```

Comment:
Some of these properties are left from copying InternalCaseStatus but not used, find out which are not necessary and remove them